product_dimension_compute_volume/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# SPDX-FileCopyrightText: 2022 Coop IT Easy SCRLfs
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Copyright are usualy removed from trivial init files.

Reply:
I am co-maintainer of [a project that advocates adding licensing blurbs to every single file](https://reuse.software) ;) I'll happily remove it here if it's unwanted, but I don't see the harm in keeping it.